src/gol.js

```diff
@@ -0,0 +1,37 @@
+export function countLiveNeighbours(board, i, j) {
```

Comment:
(Y) correct order of functions. Each function should be defined before it is used.